dojo/templates/dojo/view_risk_acceptance.html

```diff
@@ -384,4 +385,37 @@ <h3>Notes</h3>
 <script type="text/javascript">
      {% include 'dojo/snippets/risk_acceptance_actions_snippet_js.html' %}
 </script>
+<script type="text/javascript">
+    $(document).ready(function() {
+        setTimeout(function() {
+            var table = $('#findings_table').DataTable({
+                "paging": false,
+                "searching": true,
+                "ordering": true,
+                "info": false,
+                scrollCollapse: true,
+                scrollY: '30vh',
+                columns: [
+                    { title: '' },
+                    { title: 'ID' },
+                    { title: 'Severity' },
+                    { title: 'EPSS Score / Percentile' },
+                    { title: 'Name' },
+                    { 
+                        title: 'Date', 
+                        render: function(data) {
+                            var date = new Date(data);
+                            var year = date.getFullYear();
+                            var month = (1 + date.getMonth()).toString().padStart(2, '0');
+                            var day = date.getDate().toString().padStart(2, '0');
+
+                            return year + '-' + month + '-' + day;
+                        }
+                    },
+                    { title: 'Active' }
```

Comment:
I'm not sure why this is happening, but the table headings don't move with the data columns when the window is resized, leading to confusing results.

Original:
![Screenshot 2024-03-15 at 12 18 03](https://github.com/DefectDojo/django-DefectDojo/assets/1749665/39e9d421-61a4-4af1-b5bc-406f7e258b88)

Resized:
![Screenshot 2024-03-15 at 12 14 40](https://github.com/DefectDojo/django-DefectDojo/assets/1749665/035c936b-74ef-4f29-9352-c6e8f6f677c0)

Additionally, these columns differ quite a bit from the standard Finding listing template:

![Screenshot 2024-03-15 at 12 14 53](https://github.com/DefectDojo/django-DefectDojo/assets/1749665/5a37490c-d475-446e-9fc0-162df746f95d)

At a minimum I would:
* Swap Name and EPSS
* Use the same format for dates as used in the normal Finding list template

I'm also not sure ID needs to be present as a column here.

Reply:
https://github.com/DefectDojo/django-DefectDojo/pull/9737/commits/562d1eccc08b885cb480e43b39e308a40aea90d4